src/multi_stage/endogenous_retirement.jl

```diff
@@ -22,6 +22,56 @@ function get_retirement_stage(cur_stage::Int, lifetime::Int, stage_lens::Array{I
     return Int(ret_stage)
 end
 
+function compute_min_cumulative_retirements!(t::Int,inputs_d::Dict)
+
+	inputs_d[t]["dfGen"][!,Symbol("Min_Cum_Retired_Cap_MW")] = sum(inputs_d[p]["dfGen"][!,Symbol("Min_Retired_Cap_MW")] for p=1:t);
```

Comment:
To improve performance, If `t` can be very large, we could think about a way to avoid redoing the sum over all the keys `1:t` of the dictionary, since the sum over `1:t-1` is stored at `inputs_d[t-1]["dfGen"][!,:Min_Cum_Retired_Cap_MW]`. Am I correct?